bin/main.rb

```diff
@@ -1,2 +1,30 @@
-#!/usr/bin/env ruby
-puts 'Hello world!'
+require_relative '../lib/Player'
+require_relative '../lib/Game'
```

Comment:
- [ ] **Hint2:** You wrote your requires with a capital letter but your files have lower case, so it doesn't find these "capital letter files"

Reply:
DONE